acmwebsite/controllers/project.py

```diff
@@ -0,0 +1,14 @@
+"""Project controller module."""
+
+from tg import expose
+
+from acmwebsite.lib.base import BaseController
+from acmwebsite.model import DBSession, Project
+
+
+class ProjectsController(BaseController):
+    """Root controller for listing all projects"""
+
+    @expose('acmwebsite.templates.projects')
+    def index(self):
+        return dict(page='projects', projects=DBSession.query(Project).all())
```

Comment:
No need to ``.all()`` here I think... you are just iterating over it once it looks like.